dev/update-private-narhash

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Used to update the private dev flake hash reference.
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+echo "Updating $PWD/private.narHash" >&2
+
+nix flake lock ./private
+nix hash path ./private > ./private.narHash
```

Comment:
```suggestion
nix --extra-experimental-flags 'flakes nix-command' hash path ./private > ./private.narHash
```

Reply:
features, not flags.

https://github.com/nix-community/srvos/commit/a6ac04bf8ac513d8e4a506d7d69c0f161e4b528c